xsd-core/src/main/java/io/ballerina/xsdtorecordconverter/component/ComplexType.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.xsdtorecordconverter.component;
+
+import io.ballerina.xsdtorecordconverter.visitor.XSDVisitor;
+import org.w3c.dom.Node;
+
+/**
+ * Represents a complex type in the XSD schema.
```

Comment:
Dont we need the `Since` annotation.

Reply:
Fixed in [56c7e9a](https://github.com/ballerina-platform/xsd-tools/pull/1/commits/56c7e9a48dbd0d961a144bec8b2c5b011c6d5234)